locales/de.json

```diff
@@ -3057,7 +3057,8 @@
 					"pleaseSelectRole": "Bitte wähle eine Rolle aus.",
 					"selectTeacherFromRegisterOrInvitePerEmail": "Wähle eine Lehrkraft anderer Schulen aus einem zentralen Verzeichnis aus oder gib die E-Mail-Adresse an, mit der sie registriert ist. Nach Beitritt zu deinem Team kann sie Schüler:innen und Lehrkräfte ihrer Schule zum Team hinzufügen.",
 					"teacherIsFromOtherSchool": "Lehrkraft ist an einer anderen Schule",
-					"teachersRegisterProcessFinished": "Registrierung der Lehrkraft in der {{ rootThemeShortTitle }} ist abgeschlossen"
+					"teachersRegisterProcessFinished": "Registrierung der Lehrkraft in der {{ rootThemeShortTitle }} ist abgeschlossen",
+					"multipleTeachersWithEmail": "Es wurden mehrere Lehrkräfte mit dieser E-Mail gefunden. Bitte wähle die gewollte Lehrkraft aus:"
```

Comment:
"Bitte wähle die gewollte Lehrkraft aus:" if isnt a text of the requirement please ask ba.